proxy-node-acceptance-tests/features/step_definitions/user_steps.rb

```diff
@@ -53,8 +53,7 @@
   assert_text('You may have selected the wrong company. Check your emails and text messages for confirmation of who verified you.')
 end
 
-
-Then("the user should be presented with an {string} page") do |http_error_code|
+Then("the user should be presented with an error page") do
   assert_text('Sorry, something went wrong')
-  assert_text('HTTP ' + http_error_code)
+  assert_text('This may be because your session timed out or there was a system error.')
```

Comment:
We should be checking also for `Sorry, something went wrong` but we have a separate story to overhaul the acceptance tests so this is fine